[2024-10-03T20:58:37.393+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T20:58:37.408+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T15:28:21.013516+00:00 [queued]>
[2024-10-03T20:58:37.416+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T15:28:21.013516+00:00 [queued]>
[2024-10-03T20:58:37.417+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-03T20:58:37.441+0530] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): validate_data> on 2024-10-03 15:28:21.013516+00:00
[2024-10-03T20:58:37.446+0530] {standard_task_runner.py:72} INFO - Started process 12355 to run task
[2024-10-03T20:58:37.450+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_ingestion_dag', 'validate_data', 'manual__2024-10-03T15:28:21.013516+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmpgl0tncx9']
[2024-10-03T20:58:37.452+0530] {standard_task_runner.py:105} INFO - Job 120: Subtask validate_data
[2024-10-03T20:58:37.517+0530] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T15:28:21.013516+00:00 [running]> on host Nithesh-Kumar.localdomain
[2024-10-03T20:58:37.615+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_ingestion_dag' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-03T15:28:21.013516+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-03T15:28:21.013516+00:00'
[2024-10-03T20:58:37.616+0530] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T20:58:37.707+0530] {file_data_context.py:222} INFO - FileDataContext loading fluent config
[2024-10-03T20:58:37.709+0530] {config.py:187} INFO - Loading 'datasources' ->
[]
[2024-10-03T20:58:37.807+0530] {validator.py:1467} INFO - 	28 expectation(s) included in expectation_suite.
[2024-10-03T20:58:38.026+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T20:58:38.026+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T20:58:38.028+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:00, 1083.94it/s]
[2024-10-03T20:58:38.072+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:01, 43.35it/s]  
[2024-10-03T20:58:38.073+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:01, 64.34it/s]
[2024-10-03T20:58:38.104+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:01, 38.79it/s]
[2024-10-03T20:58:38.111+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 200.81it/s]
[2024-10-03T20:58:38.150+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 137.39it/s]
[2024-10-03T20:58:38.150+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 137.39it/s]
[2024-10-03T20:58:38.181+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 137.39it/s]
[2024-10-03T20:58:38.192+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 137.39it/s]
[2024-10-03T20:58:38.193+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 137.39it/s]
[2024-10-03T20:58:38.193+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 437.78it/s]
[2024-10-03T20:58:38.645+0530] {logging_mixin.py:190} INFO - Data quality check passed.
[2024-10-03T20:58:39.536+0530] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T20:58:39.548+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T20:58:39.549+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_dag, task_id=validate_data, run_id=manual__2024-10-03T15:28:21.013516+00:00, execution_date=20241003T152821, start_date=20241003T152837, end_date=20241003T152839
[2024-10-03T20:58:39.634+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T20:58:39.669+0530] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-03T20:58:39.686+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
